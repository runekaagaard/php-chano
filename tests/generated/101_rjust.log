
---- EXPECTED OUTPUT
123
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::rjust() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/101_rjust.php on line 4
---- FAILED
