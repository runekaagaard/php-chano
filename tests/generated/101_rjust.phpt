--TEST--
A generated testfile for the "rjust" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => '123')));
foreach ($items as $i) echo $i->input->rjust(4);
--EXPECT--
 123