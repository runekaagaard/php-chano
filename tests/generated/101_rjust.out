Fatal error: Call to undefined method DtlIter::rjust() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/101_rjust.php on line 4